server/controllers/system/email-polling.php

```diff
@@ -145,7 +145,7 @@ public function eraseAllEmails() {
         $mailsIds = $this->mailbox->searchMailbox('ALL');
 
         foreach($mailsIds as $mailId) {
-            $this->mailbox->deleteMail($mailId);
+            $this->mailbox->moveMail($mailId, 'INBOX.Trash');
```

Comment:
Hi @J0WI, thanks again for your contribution!
Does `INBOX.Trash` work for all email providers here?

Reply:
I'm not sure how the library deals with [RFC 6154](https://tools.ietf.org/html/rfc6154). This is probably even better:
```suggestion
            $this->mailbox->moveMail($mailId, $this->mailbox->getImapPath().'Trash');
```